### PR TITLE
Add phoenix tweet

### PR DESCRIPTION
**Pull request by IstoraMandiri**

### CODE REVIEW

**Review by TheCrowbill — COMMENTED** [review]
The tweet is fine as is.

I'm waiting on approval because an identical tweet was just approved for `eth_classic`.

**Review by brolal — APPROVED** [review]

**Comment by IstoraMandiri** [comment]
Let's just keep this on the main channel for now, but as it's approved we can merge it later at some point.

**Pull request merged**
